src/components/modals/ReconfigureFCModal.tsx

```diff
@@ -308,16 +313,22 @@ export default function ReconfigureFCModal({
       width={600}
     >
       <div>
-        <h1 style={{ marginBottom: 20 }}>
-          <Trans>Reconfigure upcoming funding</Trans>
-        </h1>
-
-        {currentFC?.duration.gt(0) && (
-          <p>
-            <Trans>
-              All changes will be applied to the <b>upcoming</b> funding cycle.
-            </Trans>
-          </p>
+        {currentFC?.duration.gt(0) ? (
+          <React.Fragment>
+            <h1 style={{ marginBottom: 20 }}>
+              <Trans>Reconfigure upcoming funding</Trans>
+            </h1>
+            <p>
+              <Trans>
+                All changes will be applied to the <b>upcoming</b> funding
```

Comment:
**suggestion (non-blocking)** - to ever so slightly improve HTML semtanics here, we could use a `<strong>` instead of `<b>`. [Here's a decent explanation why](https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em#:~:text=b%20or%20i%20means%20you,might%20use%20a%20different%20mapping.)

```suggestion
                All changes will be applied to the <strong>upcoming</strong> funding
```